packages/common/src/ui/icons/CheckboxDisabled.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export const CheckboxDisabled = (props: SvgIconProps): JSX.Element => {
```

Comment:
Maybe `CheckboxIndeterminate`? 🤔  An actual disabled checkbox I think just being a light grey of either checked or unchecked?

Reply:
Yep. good idea - I took the name from the svg I stole; but Indeterminate is more appropriate.